### PR TITLE
Added tests to back the claims in the opened issues

### PR DESCRIPTION
**Pull request by mgttlinger**

### CODE REVIEW

**Comment by mgttlinger** [comment]
A few of the things only work once the `head` -> `headOption` fix is in the branch

**Comment by timperrett** [comment]
@mgttlinger CI fails :-(

**Comment by mgttlinger** [comment]
@timperrett Well that's the point. I added tests which showcase these errors in the path search.

**Comment by timperrett** [comment]
@mgttlinger ok you didn't reference the issue so that wasn't clear. You're talking about https://github.com/Verizon/quiver/issues/21 one assumes? We can discuss on the issue about a potential solution. Thanks for finding :-)

**Comment by mgttlinger** [comment]
#21 and #22 actually

**Comment by rossabaker** [comment]
The cats port (#25) is a little more difficult than I envisioned, so I expect this and #24 can get through first, and then a subsequent Cats release after that.

**Comment by mgttlinger** [comment]
This should fix #21 and #22

File: core/src/main/scala/package.scala [hunk 1]
```diff
@@ -77,7 +77,7 @@ package object quiver {
    * Labeled path through a graph
    * @group types
    */
-  type LPath[N,A] = Adj[N,A]
+  type LPath[N,A] = (N,Vector[(N,A)])
```
Comment (timperrett) [inline core/src/main/scala/package.scala:80]: @kaiserpelagic can you just cast your eye over this to make sure it wont cause problems with nelson?

**Review by rossabaker — APPROVED** [review]
:+1:

**Comment by rossabaker** [comment]
Folded into release branch on #27.

**Pull request closed without merging**